### ISSUE TITLE
MAC APP Version cannot find Chromium Driver

### ISSUE THREAD

**Issue by kwmansfield** [jprouty/mint-amazon-tagger#65]
I am getting the following errors:
This is for the MAC app.  I am running macos 10.15.5

Logging into Amazon.com
Internal error while creating updates: Message: unknown error: cannot find Chrome binary
  (Driver info: chromedriver=2.41.578706 (5f725d1b4f0a4acbf5259df887244095596231db),platform=Mac OS X 10.15.5 x86_64)
Traceback (most recent call last):
  File "mintamazontagger/main.py", line 596, in create_updates
  File "mintamazontagger/main.py", line 640, in do_create_updates
  File "mintamazontagger/tagger.py", line 64, in create_updates
  File "mintamazontagger/orderhistory.py", line 65, in fetch_order_history
  File "mintamazontagger/orderhistory.py", line 180, in get_amzn_driver
  File "selenium/webdriver/chrome/webdriver.py", line 81, in __init__
  File "selenium/webdriver/remote/webdriver.py", line 157, in __init__
  File "selenium/webdriver/remote/webdriver.py", line 252, in start_session
  File "selenium/webdriver/remote/webdriver.py", line 321, in execute
  File "selenium/webdriver/remote/errorhandler.py", line 242, in check_response
selenium.common.exceptions.WebDriverException: Message: unknown error: cannot find Chrome binary
  (Driver info: chromedriver=2.41.578706 (5f725d1b4f0a4acbf5259df887244095596231db),platform=Mac OS X 10.15.5 x86_64)

Internal error while creating updates: Message: unknown error: cannot find Chrome binary
  (Driver info: chromedriver=2.41.578706 (5f725d1b4f0a4acbf5259df887244095596231db),platform=Mac OS X 10.15.5 x86_64)

I have tried the package off of github and from ppypi.org and get same error.  

If I am looking at things properly using "show package contents" there are only 2 files under selenium>webdriver>remote and they are getAttribute.js and isDisplayed.js.  Shouldn't webdriver.py be at this location?  I am a user not a developer so I am way over my head.  

BTW, I really appreciate this program.  It is extremely useful.  Thank you for all the work you have put in on it.

**Comment by jprouty** [comment]
Hello! @kwmansfield thanks for reaching out. Based on your last comment, it sounds like you've used the tool before (And find it useful! Woohoo); have you had it working before on this same mac?

I'm impressed you are able to get around github/Pypi. If you're interested, you could also try the prepackage binary to see if that improves anything.

Based on the error message, it's pretty clear you have a somewhat old version of the package, as it's trying to fetch a [really] old version of chromedriver (chromedriver=2.41.578706). If you're comfortable with PyPI/pip, I would recommend upgrading to the latest version and trying again:
`pip install --upgrade mint-amazon-tagger`

**Comment by kwmansfield** [comment]
I must have screwed up my Python environment.  I am getting the same behavior from the binary and cli versions.  I have multiple versions of Python do to my tinkering.  I have the original Python 2.7 that came on my mac and also 3.7 and 3.8.  I also installed an application using Conda so I have the Conda versions of Python and directories for Python 3.7 and 3.8 in the Conda directory structure.  I obviously don't expect you to help with my screwup.

I also tried installing mint-amazpn-tagger in a virtual environment thinking that might work.  Not matter what I do it can't find the path to the chrome driver.  

I have been playing around with Home Assistant (a smart home platform) on a Raspberry Pi 4 and learned just enough Python to be dangerous.  I then decided I wanted to learn more so I upgraded my database from sqllite to MySQL and was playing around with querying the database remotely with Jupyter Labs and Pandas on my MAC. 

I am an old retired engineer.  My only formal IT training was Fortran IV on an IBM mainframe with punch cards:-).  Over the years I have messed around with C, VBA and now Python and can normally hack around and get something working, or break the entire OS!
